.config/CredScanSuppressions.json

```diff
@@ -32,6 +32,10 @@
     {
       "file": "\\tests\\Shared\\TestCertificates\\testCert.pfx",
       "_justification": "Legitimate UT certificate file with private key, from dotnet/aspnetcore"
+    },
+    {
+      "file": "\\tests\\Shared\\TestCertificates\\https-dsa.pem",
```

Comment:
Did you run an internal build and something was flagging? This is just a public certificate, so I wouldn't expect this to trigger CredScan

Reply:
I was following the directions in https://github.com/dotnet/aspire/blob/main/tests/Shared/TestCertificates/README.md

Are you saying that because there's no private key here, the suppression is redundant? I'm happy to remove it.

Reply:
Oh, I see what happened. Will remove and add a little more clarity to those docs.